charts/astronomer/templates/registry/registry-networkpolicy.yaml

```
@@ -99,7 +99,7 @@ spec:
     - podSelector:
         matchLabels:
           tier: nginx
-          component: {{ ternary "cp-ingress-controller" "dp-ingress-controller" (eq .Values.global.plane.mode "unified") }}
+          component: {{ ternary "cp-ingress-controller" "dp-ingress-controller" (or (eq .Values.global.plane.mode "unified") (eq .Values.global.plane.mode "control") )}}
```

i know current code is fine, but for me this is simpler :)
```suggestion
          component: {{ ternary "dp-ingress-controller" "cp-ingress-controller" (eq .Values.global.plane.mode "data") )}}
```